features/google/src/views/ministories/dataImportingStatus.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import ReportStory from "./reportStory.jsx";
+
+class DataImportingStatusReport extends ReportStory {
+    constructor(props) {
+        super(props);
+        this._neededReports = ["importersData"];
+    }
+    get title() {
+        return "Importing status";
+    }
+
+    get reportData() {
+        return this.reports.importersData;
+    }
+
+    _renderStatus(status) {
+        return (Array.isArray(status) ? status : [status]).map(
+            (each, statusIndex) => {
+                return (
+                    <div key={statusIndex}>
+                        {each.name + (each.message ? " - " + each.message : "")}
+                    </div>
+                );
+            }
+        );
+    }
+
+    _renderSummary() {
+        return (
+            <>
+                <p>
+                    Data was read using {this.reportData.length} importers. This
```

Comment:
Shouldn't we start to insert the `i18n` strings from the beginning? Might be easier to have them from the get go, even if they are the same in English and German (or Danish), than start to add them later...

Reply:
Yes that is a good point also, since the i18n isn't set up in the google feature yet, I just hard coded some stuff to have a quick prototype. But definitely should have i18n implemented sooner rather than later